examples/google-analytics/app/routes/contact.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import { Form, useSubmit } from "remix";
```

Comment:
You shouldn't need `useSubmit` here

```suggestion
import { Form } from "remix";
```